PIPS/pip-008.md

```diff
@@ -0,0 +1,28 @@
+    PIP: 008
+    Title: On-A-Budget mode (aka Doshirak mode)
+    Author: <Kosta Korenkov>
+    Created: <2018-03-18>
+    Affects: PIP004, PIP005, PIP007
+
+
+## Simple Summary
+When we short on funds, cut our costs to bare minimum. Reduce basic income buyback and keep budgets only for funds-attracting activities.
+This PIP is basically a set of amendments to PIP004, PIP005, PIP007.
+
+## Motivation
+When we are running out of cash, our top priority should be to look for more money to sustain the project. Building positive cashflow, looking for new investments and/or grants. We cut our expenses to give us more time for that and to postpone the bankruptcy.
+
+## Applicability
+This PIP applies the next Monday 00:00 UTC after Escrow council balance becomes less than 50,000 DAI worth of ETH.
+This PIP cease to apply the next Monday 00:00 UTC after Escrow council balance has more than 80,000 DAI worth of ETH.
+
+## Specification
+- PIP004: Limit partner buyback program to buy back 60,000 PSC maximum
+- PIP007: Exclude Acebusters circle from Budget program
+- PIP007: Limit budget for Communication circle to 2000 DAI per month
+- PIP007: budgets can only be spent to reimburse expenses. If you want to earn extra money, create a bounty ([PIP005](https://github.com/parsec-labs/PIPs/blob/master/PIPS/pip-005.md)).
+- PIP005: Exclude Acebusters and R&D circles from Bounty program
```

Comment:
meh, even more discrimination. now R&D is dead too, isn't it?

Reply:
we still have basic income

Reply:
super limited, but still. What's your proposal?

Reply:
I addressed existing bounties as well just now